CHANGES.md

```diff
@@ -61,6 +61,11 @@
 - update vendored copy of cmdliner to 1.1.1. This improves the built-in
   documentation for command groups such as `dune ocaml`. (#6038, @emillon)
 
+- `(coq.theory ...)` will now depend on the `META` files for declared
+  `(plugin ...)`, so `coqdep` can see them. This enables to use the
+  new `findlib`-based loading method implemented in 8.16 (as of 8.16.0
+  tho Coq itself has some bugs preventing this to work)
```

Comment:
```suggestion
- `(coq.theory ...)` will now depend on the `META` files for declared
  `(plugin ...)`, so `coqdep` can see them. This enables to use the
  new `findlib`-based loading method implemented in 8.16.
  (Currently Coq 8.16.0 has some bugs preventing this from working).
```
and also PR num + name

Reply:
Reworded the whole thing, added the issue and PR number thanks!